ansible_ai_connect/main/settings/base.py

```diff
@@ -294,7 +300,7 @@ def is_ssl_enabled(value: str) -> bool:
         },
     },
     "handlers": {
-        "console": {"class": "logging.StreamHandler", "formatter": "simple", "level": "INFO"},
+        "console": {"class": "logging.StreamHandler", "formatter": "simple", "level": "DEBUG"},
```

Comment:
do u want `DEBUG` here? you already have it enabled for the `streaming_chat`

Reply:
Just enabling for the `streaming_chat` does not enable the `DEBUG` trace and I needed to update this console handler setting to use `DEBUG` level.  But it needs to be reset to `INFO` before merging.  I will change it back to `INFO`. Thanks.

Reply:
Change those log levels (both `console` and `streaming_chat`) to `INFO`